components/gcp-click-to-deploy/src/DeployForm.tsx

```diff
@@ -344,6 +344,26 @@ export default class DeployForm extends React.Component<any, DeployFormState> {
     // enabling, make requests to enable them, then we repeat this in a loop
     // until we have no more services left, or we try too many times.
     this._appendLine(`Getting enabled services for project ${project}..`);
+    await request(
+      {
+        body: JSON.stringify(
+          {
+            Msg: 'Echo from web app?',
```

Comment:
I would prefer the communication with the Go backend is transparent to the user. Echoes would seem strange here, we can just display a message saying "Could not reach server" or something.